tests/pkg_tinyvcdiff/fakemtd.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2022 Juergen Fitschen
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ * @ingroup  tests
+ * @brief    Fake MTD device
+ *
+ * @author   Juergen Fitschen <me@jue.yt>
+ */
+#ifndef FAKEMTD_H
+#define FAKEMTD_H
+
+#include "mtd.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Page size
+ */
+#define FAKE_MTD_PAGE_SIZE        128
+
+/**
+ * @brief Pages per sector
+ *
+ * @note FAKE_MTD_PAGE_SIZE * FAKE_MTD_PAGES_PER_SECTOR must be 512 for FatFS
+ */
+#define FAKE_MTD_PAGES_PER_SECTOR 4
+
+/**
+ * @brief Sector count
+ *
+ * @note FatFS requires more than 128 sectors
+ */
+#define FAKE_MTD_SECTOR_COUNT     256
+
+/**
+ * @brief Storage size in byte
+ */
+#define MAKE_MTD_MEM_SIZE         FAKE_MTD_PAGE_SIZE * FAKE_MTD_PAGES_PER_SECTOR * FAKE_MTD_SECTOR_COUNT
```

Comment:
```suggestion
#define MAKE_MTD_MEM_SIZE         (FAKE_MTD_PAGE_SIZE * FAKE_MTD_PAGES_PER_SECTOR * FAKE_MTD_SECTOR_COUNT)
```

Also this overflows.